src/NuGet.Clients/NuGet.Tools/xlf/Resources.de.xlf

```diff
@@ -114,7 +114,7 @@
       </trans-unit>
       <trans-unit id="ProjectMigrateErrorMessage">
         <source>Project is not eligible for migration as it's not project.json based.</source>
-        <target state="translated">Das Projekt ist nicht für die Migration geeignet, da es nicht auf project.json basiert.</target>
+        <target state="new">Project is not eligible for migration as it's not project.json based.</target>
```

Comment:
I am not sure why we are replacing a string from another locale with English in the locale-specific resource file.